sql-plugin/src/main/scala/org/apache/spark/sql/rapids/datetimeExpressions.scala

```diff
@@ -877,6 +878,66 @@ case class GpuFromUnixTime(
   override lazy val resolved: Boolean = childrenResolved && checkInputDataTypes().isSuccess
 }
 
+
+class FromUTCTimestampExprMeta(
+    expr: FromUTCTimestamp,
+    override val conf: RapidsConf,
+    override val parent: Option[RapidsMeta[_, _, _]],
+    rule: DataFromReplacementRule)
+  extends BinaryExprMeta[FromUTCTimestamp](expr, conf, parent, rule) {
+
+  override def tagExprForGpu(): Unit = {
+    val timezoneShortID = extractStringLit(expr.right).getOrElse("")
+
+    if (timezoneShortID == null) {
```

Comment:
the line above `getOrElse("")`
```suggestion
    if (timezoneShortID.isEmpty()) {
```